x-pack/legacy/plugins/infra/public/utils/typed_react.tsx

```diff
@@ -25,7 +25,14 @@ interface ChildFunctionRendererOptions<RenderArgs extends {}> {
 export const asChildFunctionRenderer = <InjectedProps extends {}, OwnProps>(
   hoc: InferableComponentEnhancerWithProps<InjectedProps, OwnProps>,
   { onInitialize, onCleanup }: ChildFunctionRendererOptions<InjectedProps> = {}
-) =>
+): ConnectedComponent<
```

Comment:
wasn't sure how to solve it in a better way, if someone has a better idea feel free to suggest or commit :)

Reply:
We are getting rid of redux for the infra plugin so I wouldn't dedicate a big effort here